veridaOne/profile/v0.1.0/schema.json

```diff
@@ -0,0 +1,167 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://common.schemas.verida.io/veridaOne/profile/v0.1.0/schema.json",
+  "name": "https://common.schemas.verida.io/veridaOne/profile/v0.1.0/schema.json",
+  "title": "Verida One Profile",
+  "titlePlural": "Verida One Profiles",
+  "description": "Configuration of a profile on Verida One",
+  "type": "object",
+  "database": {
+    "name": "profiles"
+  },
+  "allOf": [
+    { "$ref": "https://core.schemas.verida.io/base/v0.1.0/schema.json" },
+    {
+      "properties": {
+        "customLinks": {
+          "title": "Custom Links",
+          "type": "array",
+          "items": {
+            "$ref": "#/$def/customLink"
+          },
+          "minItems": 0,
+          "uniqueItems": true
+        },
+        "platformLinks": {
+          "title": "Platform Links",
+          "type": "array",
+          "items": {
+            "$ref": "#/$def/platformLink"
+          },
+          "minItems": 0,
+          "uniqueItems": true
+        },
+        "walletAddresses": {
+          "title": "Wallet Addresses",
+          "type": "array",
+          "items": {
+            "$ref": "#/$def/walletAddress"
+          },
+          "minItems": 0,
+          "uniqueItems": true
+        },
+        "featuredCollectibles": {
```

Comment:
Do we want to leave this open to have featured items in the future that aren't collectibles? We had previously discussed featured links, posts, etc.

Reply:
Ok cool.